dist/src/main/resources/hawkular.sh

```diff
@@ -19,6 +19,8 @@ set_java_opts() {
     JAVA_OPTS="$JAVA_OPTS -Xmx64m -Xms64m"
     JAVA_OPTS="$JAVA_OPTS -Dhawkular.data=${HWK_DATA}"
     JAVA_OPTS="$JAVA_OPTS -Dhawkular.configuration=${HWK_CONFIG}"
+    JAVA_OPTS="$JAVA_OPTS -Dhawkular.bind-address=$(hwk_prop 'hawkular-alerts.bind-address')"
```

Comment:
Ah, I brought some technical debt from the 1.x version.
In 2.x perhaps just changing hawkular.bind-address and hawkular.port in the ${HWK_CONFIG} is enough.
I forgot to change it.
Thanks for pointing out.